src/style.css

```diff
@@ -1 +1,29 @@
 /* styles go here */
+
+div {
+  display: block;
+  /* margin-bottom: 10px; */
+}
+
+legend {
+  /* display: block; */
+  /* margin-bottom: 10px; */
+}
+
+label {
+  /* display: block; */
+  /* margin-bottom: 10px; */
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
+
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+}
```

Comment:
1) just use only two classes `.fieldset` and `.field`
2) you can use
```
.fieldset:not(:last-child) {
  margin-bottom: 20px;
}
```
and similar to `.field`